Framework/Crystal/src/SaveIsawPeaks.cpp

```diff
@@ -127,7 +127,7 @@ void SaveIsawPeaks::exec() {
     throw std::runtime_error(
         "No instrument in PeaksWorkspace. Cannot save peaks file.");
 
-  if (bankPart != "bank" && bankPart != "WISH" && bankPart != "?") {
+  if (bankPart != "bank" && bankPart != "WISHpanel" && bankPart != "?") {
```

Comment:
I'm not suggesting we change it here but would is possible to make this more general in some way and not have to update the code for each instrument that we want to support?